dev/InfoBar/Strings/en-us/Resources.resw

```diff
@@ -117,9 +117,21 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="InfoBarCloseButtonName" xml:space="preserve">
-    <value>Close</value>
-    <comment>Automation name of the close button.</comment>
+  <data name="InfoBarCloseButtonNameSeverityError" xml:space="preserve">
```

Comment:
Here and below: @gabbybilka feel free to suggest better text that should be used to announce the respective information, those were just a short text I used to plan and test things out.